client/src/components/RequestForm.js

```diff
@@ -1,83 +1,139 @@
+  const [location, setLocation] = React.useState("");
+  const [city, setCity] = React.useState("");
+  const [arrival, setArrival] = React.useState("");
+  const [price, setPrice] = React.useState("");
+  const [text, setText] = React.useState("");
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const request = {
+      artist,
+      date,
+      location,
+      city,
+      arrival,
+      price,
+      text,
+    };
+
+    alert(JSON.stringify(request));
+  }
 
-export default function ConcertCardBig() {
   return (
-    <Form type="submit">
-      <Div>
-        <Label>Band/Künstler</Label>
-        <Input type="text" placeholder=""></Input>
-      </Div>
-      <Div>
-        <Label>Datum</Label>
-        <Input type="date" placeholder=""></Input>
-      </Div>
-      <Div>
-        <Label>Location</Label>
-        <Input type="text" placeholder=""></Input>
-      </Div>
-      <Div>
-        <Label>Stadt</Label>
-        <Input type="text" placeholder=""></Input>
-      </Div>
-      <Div>
-        <Label>Anfahrt</Label>
-        <Input type="text" placeholder=""></Input>
-      </Div>
-      <Div>
-        <Label>Ticketpreis</Label>
-        <Input type="text" placeholder=""></Input>
-      </Div>
-      <Div>
-        <Label>Dein Text</Label>
-        <Input type="text" placeholder=""></Input>
-      </Div>
+    <Form onSubmit={handleSubmit}>
+      <Wrapper>
+        <Input
+          required
+          value={artist}
+          onChange={(event) => {
+            setArtist(event.target.value);
+          }}
+        />
+        <FloatingLabel>Band/Künstler</FloatingLabel>
+      </Wrapper>
+      <Wrapper>
+        <Input
+          required
+          type="date"
+          value={date}
+          onChange={(event) => {
+            setDate(event.target.value);
+          }}
+        />
+        <FloatingLabel>Datum</FloatingLabel>
+      </Wrapper>
+      <Wrapper>
+        <Input
+          required
+          value={location}
+          onChange={(event) => {
+            setLocation(event.target.value);
+          }}
+        />
+        <FloatingLabel>Veranstaltungsort</FloatingLabel>
+      </Wrapper>
+      <Wrapper>
+        <Input
+          required
+          value={city}
+          onChange={(event) => {
+            setCity(event.target.value);
+          }}
+        />
+        <FloatingLabel>Stadt</FloatingLabel>
+      </Wrapper>
+      <Wrapper>
+        <Dropdown
+          required
+          value={arrival}
+          onChange={(event) => {
+            setArrival(event.target.value);
+          }}
+        >
+          <option value="">Wie willst du anreisen?</option>
+          <option value="auto">Auto</option>
+          <option value="öpnv">ÖPNV</option>
+          <option value="fahrrad">Fahrrad</option>
+          <option value="fuß">Zu Fuß</option>
+          <option value="mitfahrgelegenheit">Mitfahrgelegenheit</option>
+          <option value="sonstiges">Sonstiges</option>
```

Comment:
You could just use a `.map()` 🗺️  and/or pass the array to the `Dropdown` and do the `.map()` there